webpack/scenes/ContentViews/ContentViewsConstants.js

```diff
@@ -51,7 +52,7 @@ export const cvVersionPublishKey = (cvId, versionCount) => `${PUBLISH_CONTENT_VI
 export const cvVersionTaskPollingKey = cvId => `CONTENT_VIEW_VERSION_POLLING_${cvId}`;
 export const cvAddComponentKey = cvId => `${CONTENT_VIEWS_KEY}_ADD_COMPONENT_${cvId}`;
 export const cvRemoveComponentKey = cvId => `${CONTENT_VIEWS_KEY}_REMOVE_COMPONENT_${cvId}`;
-
+export const cvPackagesCompare = (versionOne, versionTwo) => `${PACKAGES_KEY}_${versionOne}_${versionTwo}`;
```

Comment:
Were you gonna change these names `cvPackagesCompare` to `cvPackagesComparison`

Reply:
@sjha4  Yeah, I was. I think I asked you about this last time. If I am not wrong, you said that I can keep all new names on the next PR. Is that so?